pkg/maps/tunnel/tunnel.go

```diff
@@ -22,9 +23,22 @@ const (
 
 var (
 	// TunnelMap represents the BPF map for tunnels
-	TunnelMap = NewTunnelMap(MapName)
+	tunnelMap     = NewTunnelMap(MapName)
+	tunnelMapInit = &sync.Once{}
```

Comment:
I would have expected the following here. What did I miss? :)

```suggestion
	tunnelMap     = *Map
	tunnelMapInit = &sync.Once{}
```

Reply:
whoops, yeah, should only be initialized in the accessor function :+1: